### PR TITLE
Add README.cn.md

### PR DESCRIPTION
**Pull request by wi1dcard**
## Purpose

To add chinese readme file.

## Changes

- Modified some sections of README.md
- Add README.cn.md
- Add `.DS_Store` into .gitignore
- No code was changed.

## Effect

Users from china will be easier to get started.

## Tests

Nothing.

## Usage

Nothing.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@
 tags
 /build/logs
 *.phar
+.DS_Store
```
Comment (morozov) [inline .gitignore]: Please remove this. It doesn't belong to the project.

Comment (wi1dcard) [reply .gitignore]: Removed :D

File: README.cn.md [hunk 2]
```diff
@@ -0,0 +1,559 @@
...
+5.3: 5.3.29, 5.3.28 ...
+```
+
+列出更多次要版本：
+
+```bash
+$ phpbrew known --more
+```
+
+刷新 PHP 发布信息：
```
[the remark this answers is not in the document]
Comment (wi1dcard) [reply README.cn.md]: 此处再三考虑，担心译为「更新」，是否会导致部分用户理解为「升级」？

File: README.cn.md [hunk 3]
```diff
@@ -0,0 +1,559 @@
...
+
+```bash
+$ phpbrew update
+```
+
+刷新旧版本（低于5.4）：
```
[the remark this answers is not in the document]
Comment (wi1dcard) [reply README.cn.md]: 此处同上。

File: README.cn.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jhdxr) [reply README.cn.md]: 好吧，能够理解。
我之前考虑的原因是`刷新`一般对应的是 `refresh`，但用`更新`的确有可能造成这样的误解

**Pull request merged**